source/classes.tex

```diff
@@ -2797,12 +2797,12 @@
 \begin{note}
 A static member function does not have a \tcode{this}
 pointer\iref{class.this}.
+A static member function cannot be declared \tcode{virtual}\iref{dcl.fct.spec}.
+There cannot be a static and a non-static member function with
+the same name, parameter-type-list, and trailing \grammarterm{requires-clause}\iref{over.load}.
+A static member function cannot not be declared \tcode{const},
```

Comment:
Uh, this has a "cannot not" bad phrasing. This has been merged already; could you post a follow-up pull request, please?